dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="A database browser for SQLite"
+DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite.\
+It is for users and developers wanting to create databases, search, and edit data. \
+It uses a familiar spreadsheet-like interface, and you don't need to learn complicated\ SQL commands."
+HOMEPAGE="http://sqlitebrowser.org/"
+COPYRIGHT="2012-2014 Martin Kleusberg"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/sqlitebrowser/sqlitebrowser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d0d2e06a69927ba1d0b955f3261ce70c61befc5bd5ddaa06752dae8bb4219ed8"
+SOURCE_FILENAME="sqlitebrowser-$portVersion.tar.gz"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	#sqlitebrowser = $portVersion
+	app:haikuporter
+
+	"
+REQUIRES="
+	haiku >= $haikuVersion
+	"
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsqlite3
+	libqt4_devel >= 4.8.0
+	devel:libphonon
+	devel:libQt3Support
+	devel:libQtCLucene
+	devel:libQtCore
+	devel:libQtDBus
+	devel:libQtDeclarative
+	devel:libQtDesigner
+	devel:libQtDesignerComponents
+	devel:libQtGui
+	devel:libQtHelp
+	devel:libQtMultimedia
+	devel:libQtNetwork
+	devel:libQtScript
+	devel:libQtScriptTools
+	devel:libQtSql
+	devel:libQtSvg
+	devel:libQtTest
+	devel:libQtWebKit
+	devel:libQtXml
+	devel:libQtXmlPatterns
+	devel:libQtUiTools
```

Comment:
On qt5 should be needed : qtgui qtnetwork qtwidgets, the rest might come as dependencies.